jmh/src/main/scala/filodb.jmh/QueryHiCardInMemoryBenchmark.scala

```diff
@@ -125,7 +125,7 @@ class QueryLeafScanInMemoryBenchmark extends StrictLogging {
   @Benchmark
   @BenchmarkMode(Array(Mode.AverageTime))
   @OutputTimeUnit(TimeUnit.MICROSECONDS)
-  @OperationsPerInvocation(500)
+  @OperationsPerInvocation(100)
   def scanSumOfRateBenchmark(): Unit = {
     (0 until numQueries).foreach { _ =>
       Await.result(scanSumOfRate.execute(store, dataset, queryConfig).runAsync, 60.seconds)
```

Comment:
You should compare results for other aggregations too. Not just sum.

Note that top-k and quantile aggregations use more memory for the accumulators.